util/afero.go

```diff
@@ -88,20 +88,30 @@ func stringify(fs afero.Fs, name string, indent []bool) (string, error) {
 // dst has a different set of contents in the same location, the behaviour is
 // currently undefined.
 // TODO: this should be made more rsync like and robust!
-func CopyFs(srcFs, dstFs afero.Fs, src, dst string, force bool) error {
-	if src == "" {
+func CopyFs(srcFs afero.Fs, dstFs afero.Fs, src string, dst string, force bool) error {
+	var srcFsLen int
+
+	if len(src) <= 1 {
+		// root directory
 		src = "/"
+		srcFsLen = 1
+	} else {
+		// todo: clean this up with function that gets parent dir?
+		src = strings.TrimSuffix(src, "/")
```

Comment:
This is looking a lot cleaner! Instead of methods from strings, you should use path.Clean(), path.Split() and path.Join() since we're working with paths.